src/components/Step.js

```diff
@@ -258,7 +262,17 @@ export default class JoyrideStep extends React.Component {
   }
 
   render() {
-    const { continuous, debug, helpers, index, lifecycle, shouldScroll, size, step } = this.props;
+    const {
+      continuous,
+      debug,
+      helpers,
+      index,
+      lifecycle,
+      shouldScroll,
+      size,
+      step,
+      nodeId,
```

Comment:
Looks like the original list is alphabetized.

Reply:
updated